errata/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+  <!--
+    The data-githubrepo attribute provides the owner/repo name on github. For W3C repositories most of those are of the
+    form 'w3c/XXX', although there are groups that have their own owner for their repository.
+  -->
+  <head data-githubrepo="w3c/rdf-canon">
+    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+    <title>Open Errata for the RDF-CANON specification published by the RCH Working Group</title>
+    <link rel="stylesheet" type="text/css" href="errata.css"/>
+    <script src="https://w3c.github.io/display_errata/assets/errata.js" type="text/javascript"></script>
+    <script src="https://w3c.github.io/display_errata/assets/toc.js" type="text/javascript"></script>
+
+    <style type="text/css">
+      .todo {
+        background-color: yellow
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <p class="banner"><a accesskey="W" href="https://www.w3.org/"><img width="72" height="48" alt="W3C" src="https://www.w3.org/Icons/w3c_home" /></a> </p>
+      <br />
+      <h1 class="title">Open Errata for the RDF-CANON Specification</h1>
+      <dl>
+        <dt>Latest errata update:</dt>
+        <dd><span id="date"></span></dd>
+        <dt>Number of recorded errata:</dt>
+        <dd><span id="number"></span></dd>
+        <dt>Link to all errata:</dt>
+        <dd><span id="errata_link"></span></dd>
+      </dl>
+
+      <section data-notoc>
+        <h1>How to Submit an Erratum?</h1>
+        <p>Errata are introduced and stored in the <a href="https://github.com/w3c/rdf-canon/issues/">issue list of the document‘s GitHub repository</a>. The workflow to add a new erratum is as follows:</p>
```

Comment:
```suggestion
        <p>Errata are introduced and stored in the <a href="https://github.com/w3c/rdf-canon/issues/">issue list of the document’s GitHub repository</a>. The workflow to add a new erratum is as follows:</p>
```

Reply:
Smart quotes are problematic in too many ways and places. Better to use the straight apostrophe instead of either curly-single-quote.
```suggestion
        <p>Errata are introduced and stored in the <a href="https://github.com/w3c/rdf-canon/issues/">issue list of the document's GitHub repository</a>. The workflow to add a new erratum is as follows:</p>
```